src/components/how-to-play.js

```diff
@@ -29,9 +29,9 @@ export class HowToPlay extends Screen {
             : this.add.image(0, -170, `${this.scene.key}.title`);
 
         if (this.theme.howToPlay) {
-            this.buttonLayout = this.setLayout(["overlayBack", "audio", "settings", "previous", "next"]);
+            this.setLayout(["overlayBack", "audio", "settings", "previous", "next"]);
         } else {
-            this.buttonLayout = this.setLayout(["home", "audio", "pause", "previous", "next", "continue"]);
+            this.setLayout(["home", "audio", "pause", "previous", "next", "continue"]);
```

Comment:
Is this logic fork still used? Isn't it always How To Play now?

Reply:
That's a good point. Looks like Alexa added it in originally - will double check with her when she gets in.

Reply:
Yeah uneeded due to new select screen :)